client/src/components/AddUserForm.js

```diff
@@ -6,71 +6,63 @@ const initialState = {
   id: "",
```

Comment:
is this because we don't actually need an initial state to add key value pairs in later?